src/modules/market/selectors/market.js

```diff
@@ -236,19 +236,21 @@ export function assembleMarket(
           ...outcomeData,
           id: outcomeId,
           marketId,
-          lastPrice: formatEther(outcomeData.price || 0, { positiveSign: false }),
+          lastPrice: outcomeData.price ? formatEther(outcomeData.price, { positiveSign: false }) : null,
```

Comment:
if outcomeData.price === 0 this will return null cuz 0 is false-y. 0 might actually be a legitimate price.